hpe3parclient/client.py

```diff
@@ -4208,3 +4208,56 @@ def resyncPhysicalCopy(self, volume_name):
         info = {'action': self.RESYNC_PHYSICAL_COPY}
         response = self.http.put("/volumes/%s" % (volume_name), body=info)
         return response[1]
+
+    def admitRemoteCopyLinks(self, targetName, port_address):
```

Comment:
Can we make port address such that iscsi/wwn and ip addresses are passed as separate arguments?